rpctest/rpc_harness.go

```diff
@@ -240,16 +240,13 @@ func (h *Harness) SetUp(createTestChain bool, numMatureOutputs uint32) error {
 		return err
 	}
 	ticker := time.NewTicker(time.Millisecond * 100)
-out:
-	for {
-		select {
-		case <-ticker.C:
-			walletHeight := h.wallet.SyncedHeight()
-			if walletHeight == height {
-				break out
-			}
+	for range ticker.C {
```

Comment:
This is an odd simplification since it actually is not semantically equivalent.  The range version will break out of the loop when the channel closes, while the old code would continuously read the zero value (and spike CPU usage).  For this particular case it looks good, but need to be careful that these static tools are actually suggesting changes that are semantically equivalent.